app/src/main/java/com/connorlay/knitgrid/models/Stitch.java

```diff
@@ -97,4 +97,20 @@ public static void generateDefaultStitches() {
                 "come!", true).save();
         new Stitch("yo", R.drawable.yo, "Yarn over", "Instructions to come!", true).save();
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
I like the use of guard clauses here, but I would avoid single-line if-statements.